website/siteConfig.js

```diff
@@ -62,6 +63,12 @@ const siteConfig = {
     secondaryColor: '#ffffff',
   },
 
+  algolia: {
+    appId: 'RK2PHC0I8T', // if you are running the crawler yourself
+    apiKey: '31b405fd1196131ecf05d2c3dd2a0543',
```

Comment:
This is public right?

Reply:
Yes they are the `search` keys is how Algolia calls them... the `write` keys are safe.